conda/recipes/libcuml/meta.yaml

```diff
@@ -67,21 +85,28 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
       run:
```

Comment:
To use the `pin_compatible` below, we need a pinning of `cuda-version` in the `host` section.
```suggestion
      host:
        - cuda-version {{ cuda_version }}
      run:
```